### PR TITLE
change imu offset method

### PR DESCRIPTION
**Pull request by MrMondrian**

### CODE REVIEW

File: catkin_ws/src/state_estimation/launch/state_estimation.launch [hunk 1]
```diff
@@ -6,10 +6,10 @@
 		<include file="$(find sensors)/launch/dvl.launch"/>
     </group>
 
-	<arg name="q_imu_auv_w" default="0.0" />
-	<arg name="q_imu_auv_x" default="0.0" />
-	<arg name="q_imu_auv_y" default="0.0" />
```
Comment (antoinedang) [inline catkin_ws/src/state_estimation/launch/state_estimation.launch:11]: think the way its currently mounted is 180 on X, not important though. Good to merge for me

**Review by antoinedang — DISMISSED** [review]

File: catkin_ws/src/sim/launch/sim.launch [hunk 2]
```diff
@@ -18,23 +18,22 @@
     <arg name="vision" default="false" />
 
     <param name="sim" value="true" />
-
+    <param name="q_imunominal_imu_w" value="1.0" />
+    <param name="q_imunominal_imu_x" value="0.0" />
+    <param name="q_imunominal_imu_y" value="0.0" />
+    <param name="q_imunominal_imu_z" value="0.0"/>
+
+    <param name="q_dvl_auv_w" value="0.0" />
```
Comment (antoinedang) [inline catkin_ws/src/sim/launch/sim.launch:26]: does this work? wouldn't the state estimation param values overwrite these ones if it's launched after

Comment (MrMondrian) [reply catkin_ws/src/sim/launch/sim.launch:26]: no state estimation only does that if the sim arg is false

Comment (antoinedang) [reply catkin_ws/src/sim/launch/sim.launch:26]: oh ok i see thats gucci then

File: catkin_ws/src/state_estimation/launch/state_estimation.launch [hunk 3]
```diff
@@ -4,40 +4,22 @@
...
-	<arg name="auv_dvl_offset_y" default="0.0" />
-	<arg name="auv_dvl_offset_z" default="-0.3" />
 
-	<node name="state_aggregator" pkg="state_estimation" type="state_aggregator.py" respawn="true" output="screen">
-		<param name="update_rate" value="100" />
-		<param name="q_imu_auv_w" value="$(arg q_imu_auv_w)" />
-		<param name="q_imu_auv_x" value="$(arg q_imu_auv_x)" />
-		<param name="q_imu_auv_y" value="$(arg q_imu_auv_y)" />
-		<param name="q_imu_auv_z" value="$(arg q_imu_auv_z)" />
+		<param name="q_imunominal_imu_w" value="1.0" />
```
Comment (antoinedang) [inline catkin_ws/src/state_estimation/launch/state_estimation.launch:8]: 180 on X for the IMU currently

**Pull request closed without merging**